invoicing_app/management/commands/generate_tax_receipts_old.py

```diff
@@ -173,7 +173,7 @@ def generate_tax_receipt_per_payment_options(self, payment_options):
                     self.generate_tax_receipt_event(payment_option, payment_option.event)
 
             except Exception as e:
-                raise self._log_exception(e)
+                print 'ERROR linea 176'
```

Comment:
It think this is wrong! You only have to raise the exception.

Reply:
done